src/Compilers/CSharp/Test/Semantic/Semantics/RefReturnTests.cs

```diff
@@ -1,1317 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
-using Microsoft.CodeAnalysis.Test.Utilities;
-using Roslyn.Test.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.CSharp.UnitTests
-{
-    public class RefReturnTests : CompilingTestBase
```

Comment:
@VSadov it looks like you ported these tests but never actually included them in the project. Hence they're not actually getting run here.